### DISCUSSION TITLE
Shared Raid Instance ID

### DISCUSSION THREAD

**Discussion by Mcadeez** [chromiecraft/chromiecraft#7194] [Suggestions]
Is it possible to implement a system where if players with different raid IDs could blend/merge together to finish one instance of the raid? 

You could make restrictions to it (if needed):
1) All players have to be in the same guild
2) All instances before the merge, have to have the same number of killed bosses
3) All instances before the merge must have at least the first boss killed
4) Other

Whatever makes it worth trying to implement.

**Comment by krutoj87** [comment]
Why restrict it to players in the same guild? This would only help guilds with multiple raid groups and PUGs would be completely screwed.

I think this suggestion is a good idea, if it's possible to be implemented. I also think this was how it worked on the OG servers. Might have been implemented later, but I don't really think so, because I didn't raid past WotLK and I'm pretty sure I've used that feature back then. Though I can be misremembering things. 
If it's implemented I wouldn't limit it to players of the same guild though. So PUGs and smaller guilds can also profit from it. 

Maybe it can be made, so the raid leaders ID is the one that gets continued and players with other IDs, that have the same amount of bosses killed, will just be switched to the raid leaders ID when they enter the raid. So they lose their old ID and continue to clear the new ID.

As a downside this has potential for Drama, when one renegade player finishes the ID of the guild raid with randoms before the second raid day of the guild. But that's already possible if you find people that don't have lockouts already and I think the chances for this to happen on CC is rather low.

**Comment by Nyeriah** [comment]
Hello and thanks for your suggestion!

It is indeed how raids started working on modern versions of the game however such system would be too complex to ensure it cannot be exploited with our current client limitations. So unfortunatelly, even though it is a rather nice idea, we cannot make a move towards it at this time, I am sorry.

Thanks once again,
Regards.
